src/errors/service-error.js

```diff
@@ -0,0 +1,2 @@
+export default class ServiceError extends Error {
```

Comment:
lets move an error message construction here:
```
  constructor(message) {
    const serviceErrorMessage = `User service error: ${message}`;

    super(serviceErrorMessage);
  }
```
So in this case you'll remove common part duplication from user service class and will only pass the error message to it:
```
throw new ServiceError(error.message)
```